omnigibson/sensors/vision_sensor.py

```diff
@@ -108,12 +110,16 @@ def __init__(
         load_config=None,
         image_height=128,
         image_width=128,
+        focal_length=17.0,
```

Comment:
`17.0` seems a bit arbitary. Is there a reason behind this magic number? If so, maybe add a comment?

Reply:
Yeah 17 is roughly the human focal length -- it's the same focal length we use for the viewer camera so I just made that the default. added a comment